File: ProjecIntegration.Api/ProjecIntegration.Api/Models/Entity/Entreprise.cs
using ProjecIntegration.Api.Models.BaseEntity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjecIntegration
{
    public class Entreprise :BaseEntity
    {
        public string Nom { get; set; }
        public string Adress { get; set; }
        public List <Complexe> Complexes { get; set; }
    }
}

File: ProjecIntegration.Api/projectintegrationFront/src/app/class/entreprise.ts
import { Complexe } from "./complexe";

export class Entreprise {
  Nom: string;
  Adress: string;
  Complexes: Complexe[]; // Assuming Complexe is another class

  constructor(Nom: string, Adress: string, Complexes: Complexe[]) {
    this.Nom = Nom;
    this.Adress = Adress;
    this.Complexes = Complexes;
  }
}
